### PR TITLE
X sp user profile property

### PR DESCRIPTION
**Pull request by camiloborges**

### CODE REVIEW

**Comment by camiloborges** [comment]
I suspect I found a problem. I was reading the code and it seems it doesn't allow having a property without a mapped connection attribute

**Comment by BrianFarnhill** [comment]
@camiloborges ping me with the details and we can chat about it

File: Modules/xSharePoint/DSCResources/MSFT_xSPUserProfileProperty/MSFT_xSPUserProfileProperty.schema.mof [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    UserOverridePrivacy = $false
+}
+
+*/
+
+
+[ClassVersion("1.0.0.0"), FriendlyName("xSPUserProfileProperty")]
+class MSFT_xSPUserProfileProperty : OMI_BaseResource
+{
+    [Key] string Name ;
```
Comment (TravisEz13) [inline Modules/xSharePoint/DSCResources/MSFT_xSPUserProfileProperty/MSFT_xSPUserProfileProperty.schema.mof]: Adding descriptions would make this easier to use in ISE

File: Modules/xSharePoint/Modules/xSharePoint.Util/xSharePoint.Util.psm1 [hunk 2]
```diff
@@ -32,6 +32,18 @@ function Get-xSharePointContentService() {
     return [Microsoft.SharePoint.Administration.SPWebService]::ContentService
 }
 
+
+function Get-xSharePointUserProfileSubTypeManager {
+    [CmdletBinding()]
+    param
+    (
+        $context
```
Comment (TravisEz13) [inline Modules/xSharePoint/Modules/xSharePoint.Util/xSharePoint.Util.psm1]: Function parameter names should be Pascal case per [style guidance](https://github.com/PowerShell/DscResources/blob/master/StyleGuidelines.md#parameter-names-should-use-pascalcase-)

File: Modules/xSharePoint/DSCResources/MSFT_xSPUserProfileProperty/MSFT_xSPUserProfileProperty.schema.mof (hunk 1 above)
Comment (BrianFarnhill) [inline Modules/xSharePoint/DSCResources/MSFT_xSPUserProfileProperty/MSFT_xSPUserProfileProperty.schema.mof]: @TravisEz13 We don't have descriptions on anything in xSharePoint yet - I'll add a backlog item to back capture this for all resources though

Comment (TravisEz13) [inline Modules/xSharePoint/DSCResources/MSFT_xSPUserProfileProperty/MSFT_xSPUserProfileProperty.schema.mof]: That is sufficient.  Definitely not blocking.

File: Modules/xSharePoint/Modules/xSharePoint.Util/xSharePoint.Util.psm1 (hunk 2 above)
Comment (TravisEz13) [inline Modules/xSharePoint/Modules/xSharePoint.Util/xSharePoint.Util.psm1]: My main concern here is that we are internally consistent, at least in a module.  It makes it easier to read and review if we all use the same naming and casing standards.

Comment (camiloborges) [inline Modules/xSharePoint/Modules/xSharePoint.Util/xSharePoint.Util.psm1]: Traviz. indeed, we are being inconsistent here :(
I will address this very one but I think we need a unit test to validate parameters are in the correct case. 
@BrianFarnhill  we might need to add this test in our 'Technical debt' backlog.

**Comment by BrianFarnhill** [comment]
@camiloborges You need a pull from dev to make this mergable again

**Comment by camiloborges** [comment]
@BrianFarnhill  go nuts

**Pull request merged**